docs/Contribute/ContributeWrite.md

```diff
@@ -0,0 +1,142 @@
+# Writing for BTCPay Server
+
+- [Writing for BTCPay Server](#writing-for-btcpay-server)
+  - [Why writing matters](#why-writing-matters)
+  - [Requirements and recommended software](#requirements-and-recommended-software)
+    - [Requirements](#requirements)
+    - [Recommended software](#recommended-software)
+  - [Contribute to the Software Stack](#contribute-to-the-software-stack)
+  - [Contribute to Documentation](#contribute-to-documentation)
+  - [Contribute to the Directory](#contribute-to-the-directory)
+  - [Review other's work](#review-others-work)
+  - [Get help from the community](#get-help-from-the-community)
+
+
+## Why writing matters
+
+By giving essential and concise informations on software, or creating these documentation pages to help wrap your mind around the software stack, writers contributing to BTCPayServer are as important as any other contributer.
+
+If you have writing skills, if you have a fair knowledge of the English language, then you can contribute to BTCPay Server or review the work of other contributors.
+
+Writing contributors can help in a number of places like the Software stack, the documentation or the directory for example.
+
+
+**Important note**: Contributions explained in this documentation are meant to be done in **English** only. If you wish to contribute in other languages, see [Translations](./ContributeTranslate.md/)
+
+## Requirements and recommended software
+
+### Requirements
+
+In order to contribute to BTCPayServer as a writer there are some requirements.
+
+Since most of the writen done in BTCPay Server is done through Github, you must have a [Github account](https://github.com/).
+It's also on Github that you can review the work done by others.
+
+### Recommended software
+
+To be able to fork repositories, create and work on branches, make Pull Requets and Issues easily, in a word, Manage your contributions, it's recommended your have [Github Desktop](https://desktop.github.com/).
+
+It is also recommended to have a Rich-Text editor.
+
+[Notepad++](https://notepad-plus-plus.org/downloads/) is a decent software and easy to use for the newer contributors.
+
+For more advanced users, [Visual Studio Code (VS Code)](https://visualstudio.microsoft.com/) is a good choice.
+The `Markdown All In One` extension in VS Code is also recommended, for visual comfort while editing and the ability to preview Markdown changes.
+
+## Contribute to the Software Stack
```

Comment:
I would remove entire `Contribute to the Software Stack` and put this in another section ,dev section of some sort, we can't expect writers to do this, nor are their contributions needed as much there.

Reply:
OK since quite a bit of time went into it, i guess it can be useful, just let's make sure what kind of work is expected in the stack. 

Also do add blog section (can help with specifics), but you can just mention we use WordPress and that contributors can even send us file for starters. Most writers know wordpress.

Reply:
For the software stack issue, see https://github.com/btcpayserver/btcpayserver-doc/pull/674#discussion_r478477194

Reply:
For the blog, could use specifics yes:

Like for example what kind of posts are suitable to send. 
Also, where should they be sent ? By E-mail ? In a PM somewhere ?

Reply:
I'll also expand on what is expected in the stack. 
Like not changing the code itself, but just rewording strings or fixing typos.